docs/reference/docs/delete-by-query.asciidoc

```diff
@@ -233,26 +233,49 @@ The JSON response looks like this:
 
 The number of milliseconds from start to end of the whole operation.
 
+`timed_out`::
+
+This flag is set to `true` if any of the requests executed during the
+delete by query execution has timed out.
+
+`total`::
+
+The number of documents that were successfully processed.
+
 `deleted`::
 
 The number of documents that were successfully deleted.
 
 `batches`::
 
-The number of scroll responses pulled back by the the delete by query.
+The number of scroll responses pulled back by the delete by query.
 
 `version_conflicts`::
 
 The number of version conflicts that the delete by query hit.
 
+`noops`::
+
+The number of documents that were ignored because the script used for
+the delete by query returned a `noop` value for `ctx.op`.
+
 `retries`::
 
-The number of retries that the delete by query did in response to a full queue.
+The number of retries attempted by delete by query. `bulk` is the number
+of bulk actions retried and `search` is the number of search actions retried.
 
 `throttled_millis`::
 
 Number of milliseconds the request slept to conform to `requests_per_second`.
 
+`requests_per_second`::
+
+The number of requests per second effectively executed during the delete by query.
+
+`throttled_until_millis`::
+
+Ask Nik. He knows, as always.
```

Comment:
Ha! It is "when will this request wake up?" So when you set `requests_per_second` it will still process a whole batch of results all at once and then it'll sleep for a while to make the indexing rate line up with what you asked for. So this is the the time, in millis since epoch, when the request will wake up and run again. Or it is 0 if we aren't throttled right now.

Honestly it isn't useful in the response because it should always be 0, it is only useful in the task output but the response and the task output are the same because that keeps my life easier.

Reply:
I added a short description for this field, pointing to the Task API. Since the field appear in the response I think we should document it.